packages/dapp/src/components/warnings/STAKETokenWarning.jsx

```diff
@@ -0,0 +1,63 @@
+import { Alert, AlertIcon, Flex, Link, Text } from '@chakra-ui/react';
+import { useBridgeContext } from 'contexts/BridgeContext';
+import { useBridgeDirection } from 'hooks/useBridgeDirection';
+import { BSC_XDAI_BRIDGE, ETH_BSC_BRIDGE } from 'lib/constants';
+import React from 'react';
+
+export const STAKETokenWarning = ({ noShadow = false }) => {
+  const { fromToken } = useBridgeContext();
+  const { bridgeDirection } = useBridgeDirection();
+
+  if (!fromToken) return null;
+  const { address } = fromToken;
+
+  let innerText = '';
+  if (
+    address.toLowerCase() ===
+      '0xb7D311E2Eb55F2f68a9440da38e7989210b9A05e'.toLowerCase() &&
+    bridgeDirection === BSC_XDAI_BRIDGE
+  ) {
+    innerText = (
+      <>
+        Bridging STAKE token from the Binance Smart Chain to the Ethereum
```

Comment:
Sorry that I was incorrect initially with the messages. Please check my comment to the original issue: https://github.com/omni/omnibridge-ui/issues/300#issuecomment-963936148.

Reply:
@akolotov I've updated the messaging as suggested, please confirm if it looks right :)